CRM/Core/Error.php

```diff
@@ -647,6 +647,23 @@ public static function createDebugLogger($prefix = '') {
   }
 
   /**
+   * Generate a hash for the logfile.
+   * CRM-13640.
+   */
+  protected static function generateLogFileHash($config) {
+    // Use multiple (but stable) inputs for hash information. TMI?
+    $md5inputs = array(
+      defined('CIVICRM_SITE_KEY') ? CIVICRM_SITE_KEY : 'NO_SITE_KEY',
+      $config->userFrameworkBaseURL,
```

Comment:
In some multi-site configurations, `userFrameworkBaseURL` may vary. Arguably, though, it's actually a good thing to separate out the logs for different frontends,

This will change the hash. If someone has a particular reporting/monitoring tool that's keyed to the old hash, they'll need to update. So we probably would want to mention this on the release changelog.

Aside: There's a similar function in `CRM_Core_Config_Runtime::getId()` which is used to generate some other files with hard-to-guess URLs (e.g. `templates_c/CachedCiviContainer.*.php`). I'm not sure if it would be good or bad to have the keys matching.